components/src/components/Map/editForm/Component.edit.data.ts

```diff
@@ -0,0 +1,122 @@
+export default {
+  key: 'customData',
+  label: 'Data',
+  weight: 20,
+  components: [
+    {
+      html: '<h2>Default Values</h2>',
+      key: 'simplecontent1',
+      type: 'simplecontent',
```

Comment:
can we use the formio version not our custom simple ones? at some point we may remove the simple ones and then we'd need to know to update this component too.

Reply:
updated in latest commit

Reply:
missed the simplecontent in Component.edit.data.ts

Reply:
Thanks for catching that!